### PR TITLE
[#20] [iOS] [UI] As a user, I can see Survey header

### PR DESCRIPTION
**Pull request by blyscuit**
## What Happened

Add Survey screen header with today information and user profile

### CODE REVIEW

File: iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveyHeader/SurveyHeaderView.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                Text("Today")
+                    .font(.boldLarge)
+                Spacer()
+                Assets.background.image
+                    .resizable()
+                    .frame(width: 36.0, height: 36.0)
+                    .cornerRadius(18.0)
+            }
+        }
+        .padding([.top, .horizontal], 20.0)
```
Comment (minhnimble) [inline iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveyHeader/SurveyHeaderView.swift:28]: I think we can move this a padding to the share constant file for later reuse 🙏

File: shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/SurveyUI.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.blisskmmic.presentation.model
+
+data class SurveyUI (
```
Comment (minhnimble) [inline shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/SurveyUI.kt]: Android naming suggestion
```suggestion
data class SurveyUiModel (
```

**Review by minhnimble — COMMENTED** [review]
rest LGTM 🚀

File: iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveyHeader/SurveyHeaderView.swift (hunk 1 above)
Comment (blyscuit) [reply iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveyHeader/SurveyHeaderView.swift:28]: Will do this when rebase with  #59 🙏

File: shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/SurveyUI.kt (hunk 2 above)
Comment (blyscuit) [reply shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/SurveyUI.kt]: Nice catch, rebase conflict. Removed.

**Review by minhnimble — APPROVED** [review]
LGTM now 🚀

**Review by suho — APPROVED** [review]
LGTM

**Pull request merged**